src/components/Editor.vue

```diff
@@ -457,9 +478,11 @@ export default {
 		onOpened({ document, session }) {
 			this.currentSession = session
 			this.document = document
-			this.readOnly = document.readOnly
+			this.documentReadOnly = document.readOnly
```

Comment:
Not a fan of having a copy of the state, can we just use document.readOnly instead in places where this is used?